src/reducer/TermItReducers.ts

```diff
@@ -95,6 +95,8 @@ function messages(state: Message[] = [], action: MessageAction): Message[] {
     case ActionType.LOGOUT:
     case ActionType.LOGIN: // Intentional fall-through
       return [];
+    case ActionType.LOGIN_KEYCLOAK: // Intentional fall-through
```

Comment:
There is no fall-through here. Either remove the comment, or modify the code so that the comment corresponds to reality.

Reply:
Sorry about that, I was copying code without much thinking I suppose